### PR TITLE
Itsadssd 32815 private design output fixes

### PR DESCRIPTION
**Pull request by uqnm**

### CODE REVIEW

File: packages/private-design-output/src/example.html [hunk 1]
```diff
@@ -357,399 +414,555 @@ <h2>Accessible accordions</h2>
...
+      <div class="uq-grid__col uq-grid__col--6"> 
+
         <h2>Cards</h2>
+        <!-- Default card -->
+        <h3>Default card</h3>
+
+        <div class="uq-grid">
+          <div class="uq-grid__col uq-grid__col--6 uq-grid__col--md-4 uq-grid__col--xl-3">
+            <div class="uq-card">
+              <div class="uq-card__image__wrapper">
```
Comment (jen-lui) [inline packages/private-design-output/src/example.html:428]: Some discrepancies with classnames here and card component, e.g. uq-card__image-wrapper, uq-card__content-wrapper, uq-card__title-super, might need to check through all the cards.

**Comment by jen-lui** [comment]
Still some discrepancies with classnames here and card component, e.g. uq-card__image-wrapper, uq-card__content-wrapper

**Review by jen-lui — APPROVED** [review]

**Review by Bcrunk — COMMENTED** [review]
So issue with Absolute images is that when the code is exported we would need to include local copies of images if we do't manually include the absolute reference. 

We don't wnat to store example images in what should be a clean code export. 
Also don't want to reference random interent images (lorempixel)

Fairly confident we don't need kitchen sink in design output, only basic page. 
everything the user needs is in the DS examples. 

only thing we lose is the ability to scan the Kitchen sink on export for bugs?

File: packages/private-design-output/src/example.html [hunk 2]
```diff
@@ -357,399 +414,547 @@ <h2>Accessible accordions</h2>
...
+        <!-- Default card -->
+        <h3>Default card</h3>
+
+        <div class="uq-grid">
+          <div class="uq-grid__col uq-grid__col--6 uq-grid__col--md-4 uq-grid__col--xl-3">
+            <div class="uq-card">
+              <a class="uq-card__link" href="#">Card title</a>
+              <div class="uq-card__image-wrapper">
+                <div class="uq-card__image">
+                  <img src="https://design-system.uq.edu.au/images/card/card-example-st-lucia-campus.jpg" alt="Card image description">
```
[the remark this answers is not in the document]
Comment (jen-lui) [reply packages/private-design-output/src/example.html:431]: What about placeholder image as src? https://via.placeholder.com/300/ccc/000/?text=placeholder%20image

**Review by Bcrunk — APPROVED** [review]

**Pull request merged**